src/services/LocalStorage.ts

```diff
@@ -14,7 +14,7 @@ const getValue = (key: string) => {
   return null
 }
 
-const getJSONValue = (key: string) => {
+export const getJSONValue = (key: string) => {
```

Comment:
can we instead export methods to set what we care about in local storage directly? this makes native mobile harder

Reply:
https://github.com/AudiusProject/audius-mobile-client/pull/198/files#diff-fe4e9763ba51e5b01b39d69396a930770d4f43999f4be50458532a34f016808eR41
cc @sliptype